pkg/sql/plpgsql/parser/plpgsql.y

```diff
@@ -335,60 +351,67 @@ opt_semi:
 | ';'
 ;
 
-pl_block: decl_sect BEGIN proc_sect exception_sect END opt_label
+pl_block: opt_block_label decl_sect BEGIN proc_sect exception_sect END opt_label
   {
-  // TODO(chengxiong): log declare section ($1) here.
-  header := $1.plpgsqlDeclareheader()
-  stmtBlock := &plpgsqltree.PLpgSQLStmtBlock{
-    Label: header.label,
-    Body: $3.plpgsqlStatements(),
-  }
-  if header.initVars != nil {
-    stmtBlock.InitVars = header.initVars
+    // TODO(chengxiong): log declare section ($1) here.
```

Comment:
I forgot why this todo was added, I think we can remove it

Reply:
Done.